fetch_bringup/config/default_controllers.yaml

```diff
@@ -55,12 +55,19 @@ base_controller:
   # use laser to only slowly collide with things
   laser_safety_dist: 1.0
 
+arm_twist:
+  type: "robot_controllers/CartesianTwistController"
+  root_name: torso_lift_link 
+  tip_name: wrist_roll_link
+
 robot_driver:
   default_controllers:
-    - "arm_controller/follow_joint_trajectory"
+    - arm_twist
     - "arm_controller/gravity_compensation"
+    - "arm_controller/follow_joint_trajectory"
     - "arm_with_torso_controller/follow_joint_trajectory"
-    - "base_controller"
```

Comment:
we shouldn't move this around -- as this is in production, we should strive to make the smallest set of changes that implement the required feature.